packages/autocomplete-plugin-tags/src/__tests__/createTagsPlugin.test.tsx

```diff
@@ -0,0 +1,519 @@
+/** @jsx h */
+import { fireEvent, waitFor, within } from '@testing-library/dom';
+import userEvent from '@testing-library/user-event';
+import { h, render, createElement, Fragment } from 'preact';
+
+import { autocomplete } from '../../../autocomplete-js';
+import { createTagsPlugin } from '../createTagsPlugin';
+
+beforeEach(() => {
+  document.body.innerHTML = '';
+});
+
+describe('createTagsPlugin', () => {
+  test('adds a tags source', async () => {
+    const container = document.createElement('div');
+    const panelContainer = document.createElement('div');
+
+    document.body.appendChild(panelContainer);
+
+    const tagsPlugin = createTagsPlugin({
+      initialTags: [
+        {
+          label: 'iPhone 12',
+        },
+      ],
+    });
+
+    autocomplete({
+      container,
+      panelContainer,
+      plugins: [tagsPlugin],
+    });
+
+    const input = container.querySelector<HTMLInputElement>('.aa-Input');
+
+    fireEvent.input(input, { target: { value: 'a' } });
+
+    await waitFor(() => {
+      expect(
+        within(
+          panelContainer.querySelector(
+            '[data-autocomplete-source-id="tagsPlugin"]'
+          )
+        ).getAllByRole('option')
+      ).toMatchInlineSnapshot(`
+        Array [
+          <li
+            aria-selected="false"
+            class="aa-Item"
+            id="autocomplete-0-item-0"
+            role="option"
+          >
```

Comment:
A change in the Autocomplete JS DOM will break this test, can we skip the `li` markup in the assertion to keep it focused to the plugin item content?

Reply:
Yes it makes sense, handled in 5f0e45b4.